dist/rudder-sdk-js/index.d.ts

```diff
@@ -174,7 +175,7 @@ declare module "rudder-sdk-js" {
    * @param name
    * @param callback
    */
-  function page(name: string, callback: apiCallback): void;
+  function page(name: string, callback?: apiCallback): void;
```

Comment:
This is not required as `page(name)` is already covered in `page(name, properties?, options?, callback?)`